pkg/alerts/policies.go

```diff
@@ -113,19 +133,103 @@ func (alerts *Alerts) UpdatePolicy(policy Policy) (*Policy, error) {
 }
 
 // DeletePolicy deletes an existing alert policy for a given account.
-func (alerts *Alerts) DeletePolicy(id int) (*Policy, error) {
+func (a *Alerts) DeletePolicy(id int) (*Policy, error) {
 	resp := alertPolicyResponse{}
 	url := fmt.Sprintf("/alerts_policies/%d.json", id)
 
-	_, err := alerts.client.Delete(url, nil, &resp)
-
+	_, err := a.client.Delete(url, nil, &resp)
 	if err != nil {
 		return nil, err
 	}
 
 	return &resp.Policy, nil
 }
 
+func (a *Alerts) QueryPolicyCreate(accountID int, policy QueryPolicyCreateInput) (*QueryPolicy, error) {
+	vars := map[string]interface{}{
+		"accountID": accountID,
+		"policy":    policy,
+	}
+
+	resp := alertQueryPolicyCreateResponse{}
+
+	if err := a.client.Query(alertsPolicyCreatePolicy, vars, &resp); err != nil {
+		return nil, err
+	}
+
+	return &resp.QueryPolicy, nil
+}
+
+func (a *Alerts) QueryPolicyUpdate(accountID int, policyID int, policy QueryPolicyUpdateInput) (*QueryPolicy, error) {
```

Comment:
Thoughts on naming mutation methods with `Mutation` prefix since it's a GraphQL mutation request? I realize the client only currently has a `client.Query()` method, which seems like it could be something more like `client.GraphQLRequest()` to be a little more semantic.

Reply:
`GraphQLRequest` seems reasonable (outside of this PR to change), Query was the direct port from the graphql client wrapper